compose-method/src/main/java/com/iluwatar/composemethod/App.java

```diff
@@ -0,0 +1,42 @@
+package com.iluwatar.composemethod;
+
+import java.util.Random;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * add original method is rough, and the new add method is using the composed method.
+ * {@link Node} is the node for list, and this term is to add the positive integer in
+ * each node's integer list
```

Comment:
Maybe first add a short explanation of the pattern and then explain how this example implements it